tests/system/test_root_metronome.py

```diff
@@ -265,7 +266,7 @@ def job_run_has_secret():
 
 
 @common.masters_exact(1)
-@pytest.mark.skip(reason="we need to wait until METRONOME-100 gets to testing/master")
```

Comment:
I guess you verified on CCM cluster with at least 10 runs that this does not fail, right? I believe the reason why this is also commented is that it fetches metronome leader and when we bounce a leader in a different test, that might not be accurate. I still think we need that leader endpoint

Reply:
@alenkacz I don't understand... this test uses the leader
`metronome_leader = shakedown.master_leader_ip()`